tests/layer_tests/pytorch_tests/test_matmul.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pytest
+import torch
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestMatMulOperation(PytorchLayerTest):
+    def _prepare_input(self, matrix, vector):
+        matrix_input = torch.tensor(matrix, dtype=torch.float32)
+        vector_input = torch.tensor(vector, dtype=torch.float32)
+        return [matrix_input, vector_input]
```

Comment:
Tests fail because you create torch tensors here, we expect that `_prepare_input` returns tuple of numpy arrays here.